### PR TITLE
Styling for form pages

### LINKED ISSUE #29: Styling for form pages

### PR DESCRIPTION
**Pull request by fanzzz368**
### ℹ️ Issue

Closes #29 

### 📝 Description

- Made the spacing between the input fields on the two forms consistent 
- Added form disclaimers at the bottom right before the submit button

### ✔️ Verification

- Viewed the forms on localhost to verify that styling is applied
- 
Grant Form:
Screenshot 2023-04-02 at 7 32 30 PM

Screenshot 2023-04-02 at 7 33 06 PM

Medical Form:
Screenshot 2023-04-02 at 7 33 31 PM

Screenshot 2023-04-02 at 7 33 46 PM

### 🏕️ (Optional) Future Work / Notes

Did you notice anything ugly during the course of this ticket? Any bugs, design challenges, or unexpected behavior? Write it down so we can clean it up in a future ticket!

### CODE REVIEW

File: apps/frontend/src/components/form/GrantForm.tsx [hunk 1]
```diff
@@ -278,6 +310,15 @@ const ExampleForm: React.FC<ExampleFormProps> = ({ onSubmit }) => (
         </FormControl>
       )}
     </Field>
+    </Box>
+    <Box py="2">
+    <Text as="i" fontSize={"xs"}>
```
Comment (rymaju) [inline apps/frontend/src/components/form/GrantForm.tsx]: ⛏️ This isn't indented properly, did you run a linter?

**Review by rymaju — COMMENTED** [review]
Good but please run a linter!

File: apps/frontend/src/components/form/GrantForm.tsx (hunk 1 above)
Comment (fanzzz368) [reply apps/frontend/src/components/form/GrantForm.tsx]: Sorry I didn't used linter earlier. Fixed!

Comment (fanzzz368) [reply apps/frontend/src/components/form/GrantForm.tsx]: wait nvm theres a merge conflict let me fix that

Comment (fanzzz368) [reply apps/frontend/src/components/form/GrantForm.tsx]: fixed

File: apps/frontend/src/components/form/GrantForm.tsx [hunk 2]
```diff
@@ -47,237 +49,278 @@ const alertOnSubmit = async (values: FormValues) => {
...
-            Requested Grant Amount (in USD)
-          </FormLabel>
-          <InputGroup>
-            <InputLeftElement
-              pointerEvents="none"
-              color="gray.300"
-              fontSize="1.2em"
-              children="$"
-            />
+    <Box py="2">
```
Comment (jackielincroft) [inline apps/frontend/src/components/form/GrantForm.tsx]: looks like "py" stands for "paddingY" as a chakra ui prop, could you change it to the full written out version just for readability?

**Review by jackielincroft — DISMISSED** [review]
looks good!!

File: apps/frontend/src/components/form/GrantForm.tsx (hunk 2 above)
Comment (fanzzz368) [reply apps/frontend/src/components/form/GrantForm.tsx]: Yes!

**Review by jackielincroft — APPROVED** [review]
LGTM!! 🥳

**Pull request merged**